src/Components/Profile/Options/Options.jsx

```diff
@@ -37,16 +37,40 @@ class Options extends React.Component {
         e.preventDefault();
 
         const name = this.state.name;
-
-        this.updateProfile('name', name);
+        if(this.validatePicture(name)){
+            this.updateProfile('name', name);
+        } else {
+            // error
+        }
     }
 
     handlePPSubmit(e) {
         e.preventDefault();
 
         const picture = this.state.picture;
+        if(this.validatePicture(picture)){
+            this.updateProfile('pic', picture);
+        } else {
+            // errors
+        }
+    }
+
+    validateName(name){
+        return name < 32;
```

Comment:
Shouldn't this use `.length` or something? Else shouldn't it be `validateNameLength(name_length)`

Reply:
lmao i'm tired ok